octgnFX/Octgn.DataNew/GameSerializer.cs

```
@@ -112,36 +114,51 @@ public object Deserialize(string fileName)
             #region gameBoards
             if (g.gameboards == null)
             {
-                var defBoard = new GameBoard();
-                defBoard.Name = "Default";
-                defBoard.Source = g.table.board == null ? null : Path.Combine(directory, g.table.board);
-                defBoard.XPos = g.table.boardPosition == null ? 0 : double.Parse(g.table.boardPosition.Split(',')[0]);
-                defBoard.YPos = g.table.boardPosition == null ? 0 : double.Parse(g.table.boardPosition.Split(',')[1]);
-                defBoard.Width = g.table.boardPosition == null ? 0 : double.Parse(g.table.boardPosition.Split(',')[2]);
-                defBoard.Height = g.table.boardPosition == null ? 0 : double.Parse(g.table.boardPosition.Split(',')[3]);
-                ret.GameBoards.Add("Default", defBoard);
+                try
+                {
+                    var position = g.table.boardPosition.Split(',');
+                    var defBoard = new GameBoard
+                    {
+                        Name = "Default",
+                        Source = Path.Combine(directory, g.table.board),
+                        XPos = double.Parse(position[0]),
+                        YPos = double.Parse(position[1]),
+                        Width = double.Parse(position[2]),
+                        Height = double.Parse(position[3])
+                    };
+                    ret.GameBoards.Add("", defBoard);
+            }
+                catch { }
```

is it not an issue for this to just ignore errors creating the gameboard?

---

I wanna say there was a reason for that but I can't remember atm